src/txs/swap/TFMSwapForm.tsx

```diff
@@ -309,12 +306,8 @@ const TFMSwapForm = ({ chainID }: { chainID: string }) => {
               options={getOptions("askAsset")}
               addonAfter={
                 <AssetReadOnly>
-                  {simulatedValue ? (
-                    <Read
-                      amount={simulatedValue}
-                      decimals={askDecimals}
-                      approx
-                    />
+                  {simulatedValue && askAsset ? (
+                    <Read denom={askAsset} amount={simulatedValue} approx />
```

Comment:
Asset symbol shouldn't display after amount in swap form:

<img width="1674" alt="Screenshot 2023-08-18 at 11 44 33 AM" src="https://github.com/terra-money/station/assets/30275692/ae14d068-5e5e-4ce1-9bc0-f2e7e1c17adf">

Reply:
fixed